### ISSUE TITLE
Bug with actor tiles in sprite editor

### ISSUE THREAD

**Issue by doomer6699** [chrismaltby/gb-studio#1640]
**Describe the bug**
This error happens when you zoom in the sprite editor to 800%, copy and paste a modification of that same sprite, go back to the editor, the actor tile editor breaks!

**To Reproduce**
1. Go to the sprite editor, zoom the sprite to 800%
2. Copy and paste a modification to that same sprite into the Sprites folder
3. Return to GBS sprite editor
4. The actor's tiles must now look bad

This happens randomly, so the instructions to make this happen may not work the first time - once the error occurs, it remains like this forever only for the Actor Tiles where this error occurs.

**Screenshots**
![Captura de pantalla_2024-11-10_13-59-04](https://github.com/user-attachments/assets/fce78ec6-56a6-4ba4-ab3a-b608a64169ba)
![Captura de pantalla_2024-11-10_13-59-32](https://github.com/user-attachments/assets/1ae872bb-8e33-4308-900c-c4ac618ec96f)

**Platform (please complete the following information):**
 - OS: Debian 12 Bookworm
 - App version GBS 4.2.0 Compilation for Saturday, November 9th

**Comment by chrismaltby** [comment]
Hi @doomer6699 based on your screenshots I think I might know what the issue was. When you were modifying the sprite image were you changing the width/height of the image file? I identified an issue where after resizing the image file it would sometimes act like the image was the previous width/height causing the sprite editor to look very similar to your screenshots.

I've pushed a fix which should now be in the latest dev build here https://github.com/chrismaltby/gb-studio/actions/runs/11816652077
